docs/community/onboard.md

```diff
@@ -7,8 +7,7 @@ Find here important resources for onboarding.
 ## For everyone
 <!---* Mailing list (self-subscribe): https://lists.e-infra.cz/wws/info/project-inveniordm
```

Comment:
Can the self-subscribe to the Mailing List be added back?

Reply:
Sure. I wonder if it was left out because it's supposed to only be for partners or for some other reason. Maybe @ntarocco knows?

Reply:
Thanks, and we can also check with @mesemus (Mirek Simek) and @mirekys (Miroslav Bauer) about the mailing list's finer controls. I believe that it can be set up in such a way that the Admins (me and @ntarocco) can check each new request to subscribe, to filter out spam accounts, etc.